### PR TITLE
Support passing a timezone for rolling dates.

### PR DESCRIPTION
**Pull request by dubiousdavid**
Support passing a timezone for rolling dates.

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>This PR has failing tests. Please alleviate the errors and commit them

<details>
  <summary>basic datemath</summary>
  <code>expected { Object (test) } to deeply equal { Object (test) }</code>
</details></td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>We were able to automatically fix some formatting issues in this PR for you!</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Could not find any browser results.</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>You reduced the total lines of code! Awesome! :+1:</td>
    </tr>
  </tbody>
</table>


### Some things that were possibly fixed:

- Code that could be fixed via the --fix flag
- Formatting that could be fixed by prettier

Take a look at this commit to see what happened in detail: a1d77c2afaf8eb499be07a4fc70dc2096ec58034


And look at this wiki page to see the reasoning behind the ESLint rules: https://github.com/smartprocure/eslint-config-smartprocure/wiki/Rules-and-Why-We-Chose-Them
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9afd19688462c4319b565474882ea186cee48de9
</p>

File: src/example-types/date.js [hunk 1]
```diff
@@ -1,39 +1,87 @@
...
+  last15Months: { from: 'now/d-15M', to: 'now' },
+  last18Months: { from: 'now/d-18M', to: 'now' },
+  last24Months: { from: 'now/d-24M', to: 'now' },
+  last36Months: { from: 'now/d-36M', to: 'now' },
+  last48Months: { from: 'now/d-48M', to: 'now' },
+  last60Months: { from: 'now/d-60M', to: 'now' },
+  lastCalendarMonth: { from: 'now-1M/M', to: 'now/M-1ms' },
+  lastCalendarYear: { from: 'now-1y/y', to: 'now/y-1ms' },
+  thisCalendarMonth: { from: 'now/M', to: 'now+1M/M-1ms' },
+  thisCalendarYear: { from: 'now/y', to: 'now' },
```
Comment (daedalus28) [inline src/example-types/date.js:41]: should this be `-1ms` like this calendar month?

Comment (dubiousdavid) [reply src/example-types/date.js:45]: This should probably be `now+1y/y-1ms` to be consistent with `thisCalendarMonth`. Thoughts?

Comment (dubiousdavid) [reply src/example-types/date.js]: I changed it to the above.

**Pull request merged**